VAMobile/src/screens/SyncScreen/SyncScreen.tsx

```diff
@@ -31,6 +31,9 @@ const SyncScreen: FC<SyncScreenProps> = () => {
   const { preloadComplete: militaryHistoryLoaded, loading: militaryHistoryLoading } = useSelector<RootState, MilitaryServiceState>((s) => s.militaryService)
   const { preloadComplete: disabilityRatingLoaded, loading: disabilityRatingLoading } = useSelector<RootState, DisabilityRatingState>((s) => s.disabilityRating)
   const { data: userAuthorizedServices, isLoading: loadingUserAuthorizedServices } = useAuthorizedServices({ enabled: loggedIn })
+  // TODO: For some reason Unit Tests cannot pick up the DowntimeFeatureTypeConstants constant
```

Comment:
I do not know why, but for whatever reason when the constants are used here the Unit Tests fail. I have spent almost 10 hours on Unit Tests alone trying to make it work. @Sparowhawk has also looked at the Unit Test and confirmed that there is something wonky going on with the tests. Eventually, yes, let's move back to the `DowntimeFeatureTypeConstants` usage (hence the `TODO` tag).

Reply:
can you create a CU ticket for that?

Reply:
Done. [GH Issue #7462](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/7462)